docs/en/ingest-management/fleet/fleet-server-scaling.asciidoc

```diff
@@ -226,7 +228,19 @@ of these operations.
 
 [discrete]
 [[agent-policy-scaling-recommendations]]
-== Policy scaling recommendations
+== Scaling recommendations
+
+**{agent} policies**
 
 A single instance of {fleet} supports a maximum of 500 {agent} policies. If more policies are configured, UI performance might be impacted.
 
+**{agents}**
+
+When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `max number of agents * 2`.
```

Comment:
when I read this through the first time I got kind of confused. in retrospect the asterisk might be ok but here are some alternatives I already typed out 


```suggestion
When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `max number of agents x 2`.
```

or

```suggestion
When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to `2(max number of agents)`.
```

or 

```suggestion
When you use {fleet} to manage a large volume (10k or more) of {agents}, the check-in from each of the multiple agents triggers an {es} authentication request. To help reduce the possibility of cache eviction and to speed up propagation of {agent} policy changes and actions, we recommend setting the {ref}/security-settings.html#api-key-service-settings[API key cache size] in your {es} configuration to 2x the maximum number of agents.
```